components/PageLink.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { Button } from '@dts-stn/service-canada-design-system'
+
+export default function PageLink(props) {
+  const router = useRouter()
+  return (
+    <>
+      <div className="pt-2 md:pt-4 my-8 border-t border-gray-light ">
+        <h2 className="font-display font-bold text-[32px] md:text-[36px]">
```

Comment:
Not Blocking - I'm not sure its a good idea to use "Arbitrary values" if at all possible - we should be striving for consistency and this won't help us do that. Assuming this comes direct from the figma designs, I would suggest reviewing the font size with our designers to ensure we build with consistency, and maybe extend the font sizes in the tailwind config like we do for colors.